### ISSUE TITLE
Issue : How to fix the issue of changing month on the DateRangePicker? 

### ISSUE THREAD

**Issue by IchrakMansour** [palantir/blueprint#3464]
I used  DateRangePicker and I want to select the date which I want, but I can't because:

When I click on any date, or any button of this DateRangePicker, the previous button is always clicked and the date which I want select is not correct because this button is clicked.

As you can see :
![Capture](https://user-images.githubusercontent.com/36265843/55556388-c1c72a00-56e7-11e9-96ad-8cba3e7cc949.PNG)

You can see my code on the sandbox: https://codesandbox.io/s/nnz3x7x1xj

I want the previous button of the month will be clicked, just when I click it not always.

How to fix that ?

**Comment by adidahiya** [comment]
huh, that's really strange. I can see the issue in your sandbox. My guess is that it has to do with the CSS loaded on the page, but even removing bootstrap CSS doesn't solve the issue...

**Comment by IchrakMansour** [comment]
Yes @adidahiya ... but when I run it without dialog, it works well, but when I integrate it with the dialog, it works like I say
